internal/boxcli/writer/writer.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2023 Jetpack Technologies Inc and contributors. All rights reserved.
+// Use of this source code is governed by the license in the LICENSE file.
+package writer
+
+import "io"
+
+type DevboxIOWriter struct {
+	W     io.Writer
+	Quiet bool
+}
+
```

Comment:
I'd suggest introducing an API like:
```
func New(cmd *cobra.Command, quiet bool) *DevboxIOWriter {
   ....
}
```
That way we control how all users construct this, and can make `W` and `Quiet` be private fields

Reply:
good suggestions, makes sense, I'll make that change